package.json

```diff
@@ -7,6 +7,7 @@
     "react-dom": "^16.9.0",
     "react-redux": "^7.1.0",
     "react-scripts": "2.1.8",
+    "react-test-renderer": "^16.9.0",
```

Comment:
testing stuff should be inside `devDependencies`, because in production mode testing dependencies are not needed.

Reply:
Okay, I probably find wrong instruction to install it. I will change this.